test/IRCTest.h

```diff
@@ -188,6 +198,7 @@ class TestModule : public CModule {
     std::vector<CClient*> vClients;
     std::vector<CChan*> vChannels;
     EModRet eAction = CONTINUE;
+    bool bSendHooks = false;
```

Comment:
I don't see a problem with indicating full list of callbacks called.
How is HALT simplified?